### ISSUE TITLE
Add a dist with separate css + js files.

### ISSUE THREAD

**Issue by euvl** [euvl/vue-js-modal#120]
Per conversation with @0pt1m1z3r, there is a need to add a build target with `js` and `css` being in separate files.

The content of `dist` will look like: 
```
/ dist
   +-- index.js
   +-- ssr.index.js
   +-- ssr.nocss.js
   +-- ssr.css
```

**Comment by lehni** [comment]
I'd appreciate a version that doesn't come with default styling also, both client and ssr.

**Comment by euvl** [comment]
hey, it should be fixed in 1.3.7, will publish it tomorrow
